src/pages/events/[eventId]/edit/index.page.tsx

```diff
@@ -13,11 +14,35 @@ export const getServerSideProps = getApplicationServerSideProps(
   async ({ req, query, queryClient, cookies }) => {
     const { eventId } = query;
 
-    await prefetchGetEventByIdQuery({
-      id: eventId,
-      req,
-      queryClient,
-    });
+    await Promise.all([
```

Comment:
I wonder if we need an Promise.all 
We don't need to fetch the event if the user hasn't permission? 
So we can probably do the redirect earlier, it will be a bit more performant I think. 

So you only do the fetch of the event if the user has the correct permissions.